src/pages/stacks-blockchain/stacking.md

```diff
@@ -1,50 +1,54 @@
+- Distribution: Collected bitcoins from the registration process are sent to a set eligible Stacks tokens holders
 
-In order to qualify for Stacking rewards, an STX holder must:
+Miners have to run software on their machines to participate in the PoX mechanism.
 
-- Hold ~94.000 Stacks tokens
-- Commit to participation before a reward cycle begins
-- Lock Stacks tokens for a lockup period
-- Specify a Bitcoin address to receive rewards
-- Vote on a Stacks chain tip
-
-## Consensus algorithm: Stacking
+[@page-reference | inline]
+| /mining
 
-- each leader candidate registers itself for a future election by sending a key transaction
-- transaction both burns the leader's cryptocurrency (proof-of-burn) and spends the leader's energy (proof-of-work), and registers the leader's preferred chain tip and new VRF seed for selection in the cryptographic sortition
-- one election across all candidate leaders (across all chain tips)
-- each correct leader candidate first selects the transactions they will commit to include their blocks as a batch, constructs a Merkle tree from them, and then commits the Merkle tree root of the batch and their preferred chain tip (encoded as the hash of the last leader's microblock header) within the commitment transaction in the election protocol.
-- leader that commits to the winning chain tip and the peers who also burn for that leader collectively share in the block's reward, proportional to how much each one burned
-- New Stacks tokens come into existence on a fork in an epoch where a leader is selected, and are granted to the leader if the leader produces a valid block
-- However, the Stacks blockchain pools all tokens created and all transaction fees received and does not distribute them until a large number of epochs (a lockup period) has passed. The tokens cannot be spent until the period passes.
+## Token holder eligibility
 
-- must determine the set of addresses that miners may validly transfer funds to. Consensus of this requires a "rewarding prepare" phase in which an anchor block and reward set are decided on
-- Progression in Stacking consensus happens over reward cycles (with a fixed length). In each reward cycle, a set of Bitcoin addresses are iterated over, such that each Bitcoin address in the set of reward addresses has exactly one Bitcoin block in which miners will transfer funds to the reward address.
-- If a miner is building off a descendant of the anchor block, the miner must send commitment funds to 5 addresses from the reward set
+Stacks token holders do not automatically receive Stacking rewards. Instead, they must:
 
-### Integrating stacking
+- Commit to participation before a reward cycle begins
+- Hold ~94.000 Stacks tokens
+- Lock up Stacks tokens for a specified period
+- Set a Bitcoin address to receive rewards
 
-Ready to integrate stacking in your wallet or exchange?
+Token holders will have to use a wallet or exchange that supports participation in Stacking.
```

Comment:
this is true, but not limited to these 2.  Stacking is done by calling a contract, so anything able to call this contract would also qualify.

Reply:
yeah, good point. at this point in the flow, the reader doesn't know yet about stacking details like that it is just a contract. let me see how to be clear but also not overwhelming